drivers/watchdog/iwdg_stm32.c

```diff
@@ -1,36 +1,23 @@
 /*
- * Copyright (c) 2016 Open-RnD Sp. z o.o.
+ * Copyright (c) 2017 RnDity Sp. z o.o.
```

Comment:
You have replaced copyright on the file, instead you should add it like you did it here: https://github.com/zephyrproject-rtos/zephyr/pull/556/commits/35cb1d6a9648e48c15a0a5dd5d8249760160920d#diff-5501313a5b4bb8f477a726ee542b86fdR4